src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/OppgaveController.kt

```diff
@@ -25,13 +28,22 @@ import org.springframework.web.bind.annotation.RestController
 class OppgaveController(
     private val oppgaveService: OppgaveService,
     private val tilgangService: TilgangService,
+    private val fagsakPersonService: FagsakPersonService,
 ) {
 
     @PostMapping("/soek")
     fun hentOppgaver(@RequestBody finnOppgaveRequest: FinnOppgaveRequestDto): FinnOppgaveResponseDto {
         return oppgaveService.hentOppgaver(finnOppgaveRequest)
     }
 
+    @PostMapping("/soek/person/{fagsakPersonId}")
```

Comment:
Kan være en `@Get`. Søket over er kun en `@Post` for å kunne sende med data, men Get blir mest riktig her 🤓

Reply:
Fikset ✅